client/coral-admin/src/actions/settings.js

```diff
@@ -21,10 +23,16 @@ export const fetchSettings = () => dispatch => {
     });
 };
 
+// for updating top-level settings
 export const updateSettings = settings => {
   return {type: SETTINGS_UPDATED, settings};
 };
 
+// this is a nested property, so it needs a special action.
+export const updateWordlist = (listName, list) => {
```

Comment:
Good question. @wyattjoh, what is the benefit of having the nested word lists that require custom code?